readme.md

```diff
@@ -173,28 +173,53 @@ $ git checkout -b factory-resource 2.0.0
 
```

Comment:
so if they're pulling code from starter, the above controller code would look a little different. The hard-coded grumbles are in in a global variable in the main `index.html` so the code just becomes: `this.grumbles = grumbles`